tests/system/Database/Live/DbUtilsTest.php

```diff
@@ -115,8 +115,14 @@ public function testUtilsOptimizeTableFalseOptimizeDatabase()
 
 		$this->setPrivateProperty($util, 'optimizeTable', false);
 
-		$this->expectException(DatabaseException::class);
-		$this->expectExceptionMessage('Unsupported feature of the database platform you are using.');
+		if ($this->db->DBDebug) {
+			$this->expectException(DatabaseException::class);
+			$this->expectExceptionMessage('Unsupported feature of the database platform you are using.');
+			$util->optimizeDatabase();
+		} else {
+			$result = $util->optimizeDatabase();
+			$this->assertFalse($result);
+		}
```

Comment:
You should test both cases, `$this->db->DBDebug` is true and false.

Reply:
I still don't understand what you mean.